packages/plugin-tickets-api/src/exporter.ts

```
@@ -506,6 +508,19 @@ const fillValue = async (
 
       break;
 
+    case "tagIds":
+      const tags = await models.Tags.find({ _id: { $in: item.tagIds || [] } });
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)